.storybook/regression-test/stories.js

```diff
@@ -19,5 +19,6 @@ export default [
   'Tag',
   'Pagination',
   'Grid',
+  'ProgressBar',
   'CircularLoader',
```

Comment:
Why this PR(progress-bar) is changing snapshots of `circularLoader` ?
(I'm mentioning it here, but look on the `png files`

Reply:
Probably because the tests are outdated with the master branch.